cli/commands/site/stop.ts

```diff
@@ -0,0 +1,56 @@
+import { __ } from '@wordpress/i18n';
+import { SiteCommandLoggerAction as LoggerAction } from 'common/logger-actions';
+import { clearSiteLatestCliPid, getSiteByFolder } from 'cli/lib/appdata';
+import { connect, disconnect } from 'cli/lib/pm2-manager';
+import { stopProxyIfNoSitesNeedIt } from 'cli/lib/site-utils';
+import { isServerRunning, stopWordPressServer } from 'cli/lib/wordpress-server-manager';
+import { Logger, LoggerError } from 'cli/logger';
+import { StudioArgv } from 'cli/types';
+
+export async function runCommand( siteFolder: string ): Promise< void > {
+	const logger = new Logger< LoggerAction >();
+
+	try {
+		const site = await getSiteByFolder( siteFolder );
```

Comment:
```suggestion
		const site = await getSiteByFolder( siteFolder, false );
```
This should fix the unit tests